corehq/blobs/tests/util.py

```diff
@@ -119,3 +120,12 @@ def __init__(self, *args):
     def clean_db(self):
         self.old_db.close()
         self.new_db.close()
+
+
+class TemporarySwiftBlobDB(TemporaryBlobDBMixin, SwiftBlobDB):
+
+    def __init__(self, config):
+        super(TemporarySwiftBlobDB, self).__init__(config)
+
+    def clean_db(self):
+        pass
```

Comment:
Should remove this method or call `super(TemporarySwiftBlobDB, self).clean_db()`